### DISCUSSION TITLE
Create default anonymous users

### DISCUSSION THREAD

**Discussion by pcmanprogrammeur** [symfony/symfony#48650] [Q&A]
Hi all,

I am using the security bundle and i would like to set a default anonymous user in session.

I saw that it was possible in Symfony 4.4 : https://symfony.com/doc/4.4/security.html#3a-authentication-firewalls like the following image shows : https://symfony.com/doc/4.4/_images/anonymous_wdt.png

Is it possible to do the same in recent version please ?

Thanks for your help.

**Comment by MatTheCat** [comment]
Hi,

anonymous users were removed with the new authentication system introduced in Symfony 5.1: https://symfony.com/doc/5.1/security/experimental_authenticators.html#authenticators-removed-anonymous

That being said it should not cause any issue; what are you trying to achieve?

**Reply by pcmanprogrammeur** [reply]
Hi @MatTheCat 

I want to log all visited pages for my users !

To make that i want a user_id in all my logs and identify all visited pages by user.

**Reply by MatTheCat** [reply]
Even if you were to leverage anonymous users, where would their ID come from?

**Reply by pcmanprogrammeur** [reply]
I want to save final users in "users" table and temporary users in "users_temp" table. Each user could have a unique id in all logs.
